CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+##2018-01-23 - Supported Release 1.3.0
```

Comment:
The header hashes should be followed by a space:
`## 2018-...`

Reply:
Dropping puppet 3 is a major bump, so this would be 2.0.0

Reply:
The module also now has metadata dependencies on stdlib and registry, which is a) a breaking change that isn't mentioned yet and b) not actually dependencies of the reboot module and should be removed.

It looks like they are added to metadata for QAELK. Testing dependencies are not module dependencies, so this should be solved another way if my conclusion is correct.